[2025-01-01T06:08:25.679+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-01T06:08:25.694+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scheduler_dag_with_cron_exp_v4.my_name scheduled__2024-12-30T09:22:00+00:00 [queued]>
[2025-01-01T06:08:25.700+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scheduler_dag_with_cron_exp_v4.my_name scheduled__2024-12-30T09:22:00+00:00 [queued]>
[2025-01-01T06:08:25.701+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2025-01-01T06:08:25.716+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): my_name> on 2024-12-30 09:22:00+00:00
[2025-01-01T06:08:25.726+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=122) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-01T06:08:25.728+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'scheduler_dag_with_cron_exp_v4', 'my_name', 'scheduled__2024-12-30T09:22:00+00:00', '--job-id', '108', '--raw', '--subdir', 'DAGS_FOLDER/scheduler_dag_with_cron_exp.py', '--cfg-path', '/tmp/tmpi4tz1rtv']
[2025-01-01T06:08:25.730+0000] {standard_task_runner.py:72} INFO - Started process 123 to run task
[2025-01-01T06:08:25.730+0000] {standard_task_runner.py:105} INFO - Job 108: Subtask my_name
[2025-01-01T06:08:25.833+0000] {task_command.py:467} INFO - Running <TaskInstance: scheduler_dag_with_cron_exp_v4.my_name scheduled__2024-12-30T09:22:00+00:00 [running]> on host 2eb328b59189
[2025-01-01T06:08:26.277+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='tanzid' AIRFLOW_CTX_DAG_ID='scheduler_dag_with_cron_exp_v4' AIRFLOW_CTX_TASK_ID='my_name' AIRFLOW_CTX_EXECUTION_DATE='2024-12-30T09:22:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-30T09:22:00+00:00'
[2025-01-01T06:08:26.279+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-01T06:08:26.293+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-01T06:08:26.295+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo My Name is Md. Tanzid Hasan']
[2025-01-01T06:08:26.304+0000] {subprocess.py:99} INFO - Output:
[2025-01-01T06:08:26.313+0000] {subprocess.py:106} INFO - My Name is Md. Tanzid Hasan
[2025-01-01T06:08:26.313+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-01T06:08:26.335+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-01T06:08:26.336+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=scheduler_dag_with_cron_exp_v4, task_id=my_name, run_id=scheduled__2024-12-30T09:22:00+00:00, execution_date=20241230T092200, start_date=20250101T060825, end_date=20250101T060826
[2025-01-01T06:08:26.389+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-01T06:08:26.400+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-01T06:08:26.401+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
